cluster_experiments/experiment_analysis.py

```diff
@@ -97,6 +121,40 @@ def analysis_standard_error(
         """
         raise NotImplementedError("Standard error not implemented for this analysis")
 
+    def analysis_confidence_interval(
+        self,
+        df: pd.DataFrame,
+        alpha: float,
+        verbose: bool = False,
+    ) -> ConfidenceInterval:
+        """
+        Returns the confidence interval of the analysis. Expects treatment to be 0-1 variable
+        Arguments:
+            df: dataframe containing the data to analyze
+            alpha: significance level
+            verbose (Optional): bool, prints the regression summary if True
+        """
+        raise NotImplementedError(
+            "Confidence Interval not implemented for this analysis"
+        )
+
+    def analysis_inference_results(
```

Comment:
Renaming suggestion: change all inference_results / InferenceResults to scorecard

Reply:
which one do you like more? I'm thinking I'd like to have scorecard somewhere, not sure this is the best place